Client/core/Graphics/CRenderItem.EffectTemplate.cpp

```diff
@@ -351,6 +360,9 @@ void CEffectTemplate::CreateUnderlyingData(const SString& strFile, const SString
         }
     }
 
+    // Copy macros
+    m_Macros = macros;
```

Comment:
Not sure if its used anywhere else, but `std::move(macros)`.
Pass `std::vector` by value until here, so you can std::move it(a vector is like 8 bytes @ 32 bit, its worth the effort probably, if you can std::move a few strings in the end)

Reply:
Opps. I forgot to delete this.